<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="vmControlsGroup">
      <property name="font">
       <font>
        <family>Segoe UI Emoji</family>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="vmStatusLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>458</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Trạng thái VM: Đang chờ...</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>190</y>
         <width>458</width>
         <height>28</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QPushButton" name="launchButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>458</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Launch</string>
       </property>
      </widget>
      <widget class="QPushButton" name="viewLogButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>458</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>View Log</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
